src/package.json

```diff
@@ -10,7 +10,7 @@
     "pre-install": "git submodule update -i -r"
   },
   "dependencies": {
-    "@json2csv/node": "6.1.2",
+    "@json2csv/plainjs": "^7.0.1",
```

Comment:
Remove the caret for all the packages whenever you find it. You should never add the cater symbol.

[@shubham](https://github.com/shubhambansaltarento)